spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_structural_static.py

```diff
@@ -51,100 +53,84 @@ class SynapseDynamicsStructuralStatic(AbstractSynapseDynamicsStructural,
                 )
             )
 
-
-    :param f_rew: Frequency of rewiring (Hz). How many rewiring attempts will\
-        be done per second.
+    :param partner_selection: The partner selection rule
+    :param formation: The formation rule
+    :param elimination: The elimination rule
+    :param f_rew: How many rewiring attempts will be done per second.
     :type f_rew: int
-    :param weight: Initial weight assigned to a newly formed connection
-    :type weight: float
-    :param delay: Delay assigned to a newly formed connection
-    :type delay: int
+    :param initial_weight: Weight assigned to a newly formed connection
+    :type initial_weight: float
+    :param initial_delay: Delay assigned to a newly formed connection
+    :type initial_delay: int or (int, int)
```

Comment:
if synaptic rewiring can run at `0.1 ms` timestep should delay be float so that it can do the minimum delay of 0.1?

Reply:
Fixed; this also reminded me that we weren't doing this correctly at 0.1ms, as the delay wasn't scaled by the timestep (oops).